src/main/java/common/TableUtil.java

```diff
@@ -0,0 +1,23 @@
+package common;
+
+import jakarta.persistence.Table;
+
+public class TableUtil {
+    private final String name;
```

Comment:
```suggestion
    private final String name;
    private final List<TableColumnAttribute> tableColumnAttributes;
```
entity을 관리한다고 따지면 entity 하위에 있는 column 정보도 그 대상이 되기 때문에 이렇게 설계할 수도 있습니다 :) 
너무 어렵게 생각하지마시고 객체간의 정보결합을 어떻게할지 고민해보시면 좋을 것 같아요 😄

Reply:
진영님 질문이 있습니다. ( TableUtil > TableName으로 변경한다고 하겠습니다. )
tableColumnAttributes 리스트를 공유(?) 를 해야하는게 아닌가요..? TableColumnAttribute 클래스가 가지고 있는 tableColumnAttributes와 TableName 클래스가 가지고 있는 tableColumnAttributes 필드를 말씀드리는겁니다!

Reply:
구조가 많이 꼬여있는데 단순하게 생각하면, `TableColumnAttribute`가 `TableColumnAttributes`를 가질 필요가 있나요? 
구조는
- Entity을 관리하는 객체가 있다.
- Entity을 관리하는 객체는 아래 정보를 가진다.
    - table의 이름을 가진다.
    - Entity가 보유하고 있는 Column들의 정보를 가진다.
    - 각 Column은 서로 연관관계가 있을 필요가 없다 (TableColumnAttribute가 TableColumnAttributes를 가질 필요가 없음)

이라면
```java
public class EntityClass {
    private final tableName;
    private final TableColumnAttributes tableColumnAttributes;
}

public class TableColumnAttributes {
    private final List<TableColumnAttribute> tableColumnAttributes
}

public class TableColumnAttribute {
        private String columnName;
    private String columnDataType;
    private boolean isPrimeKey;
    private boolean isNotNull;
    private boolean isAutoIncrement;
    private boolean isTransient;
    private String fieldValue;
}
```
2단계부터 의도하고 있는 내용은 위와 같아요